WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -1,257 +1,293 @@
-            <li><a href="<%= organizations_path %>"><%= ui_icon('flag') %> <%= t '.organizations' %></a></li>
-            <li class="divider"></li>
-            <li><a href="<%= faq_path %>"><%= ui_icon('question circle') %> <%= t '.faq' %></a></li>
-            <li><a href="<%= education_path %>"><%= ui_icon('graduation cap') %> <%= t '.education' %></a></li>
-            <li><a href="<%= contact_website_path %>"><%= ui_icon('envelope') %> <%= t '.contact' %></a></li>
-            <li><a href="<%= media_instagram_path %>"><%= ui_icon('camera') %> <%= t '.media_submission' %></a></li>
-            <li><a href="<%= privacy_path %>"><%= ui_icon('user secret') %> Privacy</a></li>
-            <li><a href="<%= disclaimer_path %>"><%= ui_icon('exclamation circle') %> Disclaimer</a></li>
-            <li class="divider"></li>
-            <li><a href="<%= score_tools_path %>"><%= ui_icon('wrench') %> <%= t '.tools' %></a></li>
-            <li><a href="<%= logo_path %>"><i><%= image_tag('wca_logo.svg', width: '16px') %></i> <%= t '.logo' %></a></li>
-          </ul>
-        </li>
-        <li class="dropdown <%= Set['competitions', 'delegate_reports', 'results_submission'].include?(params[:controller]) ? 'active' : '' %>">
-          <a href="" class="dropdown-toggle top-nav" data-toggle="dropdown" data-hover="dropdown">
-             <%= ui_icon('cube') %> <%= t '.competitions' %> <span class="caret"></span>
-          </a>
-          <ul class="dropdown-menu" role="menu">
-            <li><a href="<%= competitions_path %>"><%= ui_icon('globe') %> <%= t '.all' %></a></li>
-            <li><a href="<%= my_comps_path %>"><%= ui_icon('list') %> <%= t '.my_competitions' %></a></li>
-            <% if current_user&.can_create_competitions? %>
-              <li><a href="<%= new_competition_path %>"><%= ui_icon('plus') %> <%= t '.new_competition' %></a></li>
-            <% end %>
-          </ul>
-        </li>
-        <li class="dropdown <%= Set['results', 'persons', 'media'].include?(params[:controller]) ? 'active' : '' %>">
-          <a href="" class="dropdown-toggle top-nav" data-toggle="dropdown" data-hover="dropdown">
-            <%= ui_icon('list ol') %>
-            <span class="hidden-sm"><%= t '.results' %></span>
+  <div class="brand">
+    <a href="<%= root_path %>" ><%= image_tag "WCA Logo 2020.svg" %></a>
+  </div>
+  <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+    <span class="icon-bar"></span>
+  </button>
+  <div class="navbar-collapse collapse disabled">
+    <ul class="nav navbar-nav">
+      <li class="top-nav logo">
+        <a href="<%= root_path %>"><%= image_tag "WCA Logo 2020.svg" %></a>
+      </li>
+      <li class="dropdown <%= Set['static_pages', 'posts'].include?(params[:controller]) ? '' : '' %>">
```

Comment:
With this change navigating to any information page doesn't actually highlight the menu option, I think this should do fine:

```erb
<li class="dropdown <%= (params[:controller] == 'static_pages' && params[:action] != 'homepage') ? 'active' : '' %>">
```

Reply:
Did you try that?

Reply:
Oh sorry, I didn't read carefully. Will add it now.